scene/gui/rich_text_label.cpp

```diff
@@ -4056,6 +4129,8 @@ void RichTextLabel::_bind_methods() {
 	BIND_ENUM_CONSTANT(ITEM_WAVE);
 	BIND_ENUM_CONSTANT(ITEM_TORNADO);
 	BIND_ENUM_CONSTANT(ITEM_RAINBOW);
+	BIND_ENUM_CONSTANT(ITEM_FGCOLOR);
+	BIND_ENUM_CONSTANT(ITEM_BGCOLOR);
```

Comment:
Those two should be swapped to match the enum order, otherwise the enum values are wrongly ordered in the docs.

Reply:
I swapped them in both `doc/classes/RichTextLabel.xml` and `scene/gui/rich_text_label.cpp` , it now matches the order in the `enum` list